### DISCUSSION TITLE
Rustdesk does not work on another computer (only Windows) with the same ID/relay server data.

### DISCUSSION THREAD

**Discussion by yozgatliahmed** [rustdesk/rustdesk#7196] [Q&A] [answered]
I installed Rustdesk as per the instructions on my VPS. I can reach my own computer by telephone. The mediation service is also ready. Now I wanted to reach another computer and entered the same data in the ID/relay server. However, no connection is established. Where is the problem?

![2024-02-19 17_20_29-Clipboard](https://github.com/rustdesk/rustdesk/assets/17163918/32760cfc-4a49-45c0-a166-b32b2465b918)
![2024-02-19 17_21_09-Clipboard](https://github.com/rustdesk/rustdesk/assets/17163918/4d609191-0d92-4613-b4ed-8cc43419f4b0)

**Comment by yozgatliahmed** [comment]
I tested it on my Ubuntu. Worked. But why not on the one Windows 10 computer?

**Comment by yozgatliahmed** [comment]
When I try to establish a connection on the problem Windows 10 computer, I get this error message.

![2024-02-20 13_29_07-Ubuntu Linux Station – Mozilla Firefox](https://github.com/rustdesk/rustdesk/assets/17163918/18453de2-8626-47a9-95f0-caecbc724c71)

**Comment by yozgatliahmed** [comment]
I was there today and was able to connect to the same network using my smartphone. Why doesn't this work externally?

**Comment by xlionjuan** [comment]
Can it Ping to your server

**Comment by yozgatliahmed** [comment]
Yes, on the Windows 10 computer I can ping my relay server via PowerShell.

**Reply by xlionjuan** [reply]
Try latest nightly

**Reply by yozgatliahmed** [reply]
What this mean? thx.

**Reply by xlionjuan** [reply]
Goto releases, find nightly

**Reply by yozgatliahmed** [reply]
I have installed. but same Problem. (rustdesk-1.2.4-x86_64.exe)

**Comment by yozgatliahmed** [comment]
I installed on my 4th PC (Windows 11). Its function. (Remotely too).
What is the Problem on the Windows 10 PC? Teamviewer is function, but not Rustdesk. Interesting.

**Reply by xlionjuan** [reply, marked as answer]
Try completely uninstall RustDesk and reinstall again

And don't forget to remove RustDesk folder at `%APPDATA%`

**Reply by yozgatliahmed** [reply]
Thank you. Now its function.
